client/OPUNetTransportLayer.h

```diff
@@ -100,7 +101,7 @@ class OPUNetTransportLayer : public NetTransportLayer
 	SOCKET hostSocket;
 	int forcedPort;
 	// Peer Info
-	PeerInfo peerInfo[MaxRemotePlayers];
+	std::array<PeerInfo, 6> peerInfo;
```

Comment:
This should be using `MaxRemotePlayers` instead of the hardcoded constant `6`.

It may be necessary to change the definition of `MaxRemotePlayers` from `const` to `constexpr`.